storage/pkg/supported-digests/algorithm.go

```diff
@@ -0,0 +1,32 @@
+package supporteddigests
+
+import (
+	"fmt"
+
+	"github.com/opencontainers/go-digest"
+	"github.com/sirupsen/logrus"
+)
+
+var digestAlgorithm = digest.Canonical // Default to SHA256
```

Comment:
If this is going to be called from concurrent contexts, e.g. the c/image/copy per-layer code in `computeDiffID`, we need to protect this against write-vs-read races, e.g. using a `sync.Mutex` (well, a ~concurrent write would still be _semantically_ questionable but at least we would rule out races when storing pointers/sizes and undefined access to memory).